e2e-tests/cypress/support/appellant-submission-planning-application-number/confirmPlanningApplicationNumberHasNotUpdated.js

```diff
@@ -1,5 +1,5 @@
 module.exports = () => {
-  cy.visit('/appellant-submission/application-number');
+  cy.visit('/appeal-householder-decision/application-number');
```

Comment:
please can the dir here be updated to reflect `appeal-householder-decision` - thanks